From: Hilko Bengen <bengen@debian.org>
Date: Wed, 28 Mar 2018 16:15:11 +0200
Subject: Add Debian-specific paths for ClamAV configuration files

---
 lenses/clamav.aug | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lenses/clamav.aug b/lenses/clamav.aug
index 12bb104..8c1d6f3 100644
--- a/lenses/clamav.aug
+++ b/lenses/clamav.aug
@@ -49,5 +49,8 @@ let filter = (incl "/etc/clamd.conf")
             . (incl "/etc/freshclam.conf")
             . (incl "/etc/clamd.d/*.conf")
             . (incl "/etc/clamav/*.conf")
+            . (incl "/etc/clamav/clamd.conf")
+            . (incl "/etc/clamav/freshclam.conf")
+            . (incl "/etc/clamav/clamd.d/*.conf")
 
 let xfm = transform lns filter
